litmus-portal/frontend/src/components/CustomBreadCrumbs/index.tsx

```diff
@@ -12,19 +13,22 @@ const CustomBreadcrumbs: React.FC<CustomBreadcrumbsProps> = ({ location }) => {
   const path: string[] = location.split('/');
   let intermediatRoutes: string = '/';
   const classes = useStyles();
+  useEffect(() => {}, [path]);
+
   return (
-    <Breadcrumbs
-      separator={<NavigateNextIcon fontSize="small" />}
-      aria-label="breadcrumb"
-    >
-      <Link to="/" className={classes.breadCrumb}>
-        Home
-      </Link>
+    <Breadcrumbs aria-label="breadcrumb">
+      <Link to="/" className={classes.breadCrumb} />
       {path.map((p: string) => {
```

Comment:
it should be `paths.map((path: string))`

Reply:
don't put single character variable

Reply:
Changed.